app/models/setting.rb

```diff
@@ -92,13 +92,8 @@ def self.humanized_category
 
   def self.[](name)
     name = name.to_s
-    cache_value = Setting.cache.read(name)
-    if cache_value.nil?
-      value = find_by(:name => name).try(:value)
-      Setting.cache.write(name, value)
-      return value
-    else
-      return cache_value
+    cache.fetch("settings/#{name}") do
```

Comment:
Would it be a good practice to put the cache key in a function? Now we're repeating magic strings in our code. At least I'd expect a `CACHE_KEY_PREFIX` constant

Reply:
added a method, though tbh the tests seem to test that rails caching works...